app/models/label/LabelTable.scala

```diff
@@ -89,6 +90,8 @@ object LabelTable {
   val labelsWithoutDeleted = labels.filter(_.deleted === false)
   val neighborhoods = regions.filter(_.deleted === false).filter(_.regionTypeId === 2)
 
+  val labelsWithoutDeletedOrOnboarding = labelsWithoutDeleted.filterNot(_.gsvPanoramaId inSet GSVOnboardingPanoTable.getOnboardingPanoIds)
```

Comment:
Also what is the purpose of this one?

Reply:
It just filters out the labels placed during onboarding (well, the labels placed on the panoramas that are used during onboarding, so almost certainly those are labels from onboarding). I'm not including the tutorial labels in the label count when I compute labeling frequency.

Reply:
Gotcha, thanks!